packages/app-project/src/screens/ProjectAboutPage/ProjectAboutPageConnector.spec.js

```diff
@@ -0,0 +1,30 @@
+import { shallow } from 'enzyme'
+import React from 'react'
+
+import ProjectAboutPageConnector from './ProjectAboutPageConnector'
+import ProjectAboutPage from './ProjectAboutPage'
+
+describe('Component > ProjectAboutPageConnector', function () {
+  let wrapper
+  const testStore = {
+    store: {
+      project: {
+        about_pages: [ { id: '1234', title: 'Title', url_key: 'science_case' } ],
+        inBeta: false
+      }
+    }
+  }
+  before(function () {
+    wrapper = shallow(<ProjectAboutPageConnector testStore={testStore} pageType='science_case' />)
+  })
```

Comment:
Rather than adding props just to get tests to pass, another option would be to use sinon to stub out the `useContext` hook to return a mocked store. Neither is ideal, but I've tended to use the stub approach. For this to work, sinon expects to stub object properties, so you can't use the destructured import for `useContext` and it'll have to change to `React.useContext` in the component.

Then, in the spec file, you can setup and tear down your stub for the describe block it's being used for:
```js
let useContextStub
before(function () {
  useContextStub = sinon.stub(React, 'useContext').callsFake(() => {
    return {
      store: {
        project: {
          about_pages: [ { id: '1234', title: 'Title', url_key: 'science_case' } ],
          inBeta: false
        }
      }
    }
  })
})

after(function () {
  useContextMock.restore()
})
 ```
 
 Here's the docs ref:
> var stub = sinon.stub(object, "method");
> Replaces object.method with a stub function. An exception is thrown if the property is not already a function.
> The original function can be restored by calling object.method.restore(); (or stub.restore();).

https://sinonjs.org/releases/v9.2.4/stubs/